rfc8252_http_server/src/lib.rs

```diff
@@ -0,0 +1,395 @@
+use actix_web::{
+    self, dev::ServerHandle, http, middleware, rt, web, App, HttpRequest, HttpResponse, HttpServer,
+    Result,
+};
+use askama::Template;
+use std::result::Result as StdResult;
+use std::{collections::HashMap, sync::mpsc, thread};
+use tokio::sync::mpsc as tokio_mpsc;
+
+const DEFAULT_REDIRECT_PORT: u16 = 27097;
+#[cfg(test)]
+const DEFAULT_REDIRECT_URI: &str = "http://localhost:27097/redirect";
+
+#[derive(Template)]
+#[template(path = "OIDCAcceptedTemplate.html")]
```

Comment:
Just a thought on the templates. It looks like there is [template inheritance](https://djc.github.io/askama/template_syntax.html#template-inheritance), so one template could potentially serve both Accepted and Error. Additionally, would it be possible to use `include_str!` for the css to keep the templates small?

Reply:
Nah, I took this from what Anna provided. The idea was to use minimal changes.

Reply:
So it's not possible or we don't want to do it?

Reply:
Fair enough, though I don't think they need to be the same layout between as long as the output is the same.